LayoutTests/accessibility/visible-character-range-width-changes.html

```diff
@@ -1,3 +1,4 @@
+<!-- webkit-test-runner [ CSSScrollAnchoringEnabled=false ] -->
```

Comment:
It would be preferable to use the CSS property, although I would guess it's not implemented yet.

Reply:
I guess your commit does implement part of it, so it might be possible here.